lg_volume_control/scripts/volume_control.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import rospy
+from lg_volume_control import VolumeControl
+from std_msgs.msg import UInt8, Int8
+
+
+def main():
+    rospy.init_node('volume_control')
+
+    level_change = rospy.Publisher('/volume/level', UInt8, latch=True, queue_size=1)
+    default_sink = rospy.get_param('~default_sink', '0')
```

Comment:
Need for hardware-specific configuration may be avoided by using the special `@DEFAULT_SINK@` name.  Also the README does not correctly describe the default value.